buildsystem/codecompliance/authors.py

```diff
@@ -6,6 +7,45 @@
 
 import re
 
+def get_email_obfuscation_substitutions():
+    """
+    What tokens in email should be replaced with
+    """
+    return {
+    '.' : ' noot ',
+    '@' : ' à ',
+    'gmail' : '!gm!',
+    'com' : '!corn!',
+    }
+
+def get_email_deobfuscation_substitutions():
+    """
+    Reverse map of get_email_obfuscation_substitutions
+    """
+    return dict((reversed(item) for item in get_email_obfuscation_substitutions().items()))
+
+def replace_string_with_each_kvp(string, replacements):
+    for key, value in replacements.items():
+        string = string.replace(key, value)
+    return string
+
+def obfuscate_email(email):
+    """
+    `copying.md` is particularly searchable, so let's enable contributors to hide their email from crawlers.
+    Yes, I realize that our emails are already upon our every commit. This is better than nothing.
+
+    obfuscate_email('pingu.pengy+fish@gmail.com') = pingu noot pengy+fish à !gm! noot !corn!
+    """
+    return replace_string_with_each_kvp(email, get_email_obfuscation_substitutions())
```

Comment:
since addresses are added into `copying.md` manually, I guess if the syntax is predictable enough we don't need a programmatic way to do this. certainly it's unused in the code.